plugin.json

```
@@ -43,7 +43,7 @@
                 "key": "EncryptionSecret",
                 "display_name": "Encryption Secret:",
                 "type": "generated",
-                "help_text": "The secret key used to encrypt and decrpyt OAuth token."
+                "help_text": "The secret key is used to encrypt and decrypt the OAuth token.\nRegenerating the secret key will cause the loss of data."
```

```suggestion
                "help_text": "The secret key used to encrypt and decrypt the OAuth token.\nRegenerating the secret key will cause the loss of data."
```

is this text verified and updated as per Mike suggestion? if not pls check with him and get it updated